src/service_modules/generator.ts

```diff
@@ -0,0 +1,17 @@
+export class generator{
+    getUserId(counter: number):string{
```

Comment:
this is good but try to think if we can simplify to use UUID to generate random userid :)